app/src/lib/git/core.ts

```diff
@@ -367,6 +367,8 @@ function getDescriptionForError(error: DugiteError): string | null {
       // be used as the error message (or stdout if stderr is empty), i.e. the
       // same behavior as before the ConfigLockFileAlreadyExists was added
       return null
+    case DugiteError.RemoteAlreadyExists:
+      return null
```

Comment:
this is essentially a pass-through here ie doesn't change any behavior

Reply:
Ah gotcha, this is in preparation for https://github.com/desktop/desktop/issues/9048